rewrite-gradle/src/test/java/org/openrewrite/gradle/ChangeDependencyTest.java

```diff
@@ -176,10 +176,47 @@ implementation platform("org.apache.commons:commons-lang3:3.11")
         );
     }
 
+    @Test
+    void worksWithGString() {
+        rewriteRun(
+          spec -> spec.recipe(new ChangeDependency("commons-lang", "commons-lang", "org.apache.commons", "commons-lang3", "3.11.x", null, null)),
+          buildGradle(
+            """
+              plugins {
+                  id "java-library"
+              }
+              
+              repositories {
+                  mavenCentral()
+              }
+              
+              def version = '2.6'
+              dependencies {
+                  implementation platform("commons-lang:commons-lang:${version}")
+              }
+              """,
+            """
+              plugins {
+                  id "java-library"
+              }
+              
+              repositories {
+                  mavenCentral()
+              }
+
+              def version = '2.6'
```

Comment:
Should we schedule an after visit to delete this if it's unused?

Reply:
I would prefer to close this PR as it is right now, and add extra features later in another one, because right now other repos builds are failing right now.